### PR TITLE
Update Demo app

### PR DESCRIPTION
**Pull request by Samsunggora**
Enable dark mode in Home screen 
Refactor class components to functional
Update body.css in demo app
Add dynamic configuration section to index.html

### CODE REVIEW

**Comment by Samsunggora** [comment]
image

File: demo/src/index.js [hunk 1]
```diff
@@ -23,9 +23,15 @@ let ConfigDefaults = {
 	sidebarItemsOrder: ["Table", "Home"]
 };
 
+
```
Comment (petrKavulok) [inline demo/src/index.js:26]: i believe this extra whitespace is not necessary here

File: demo/public/body.css [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+
+    overflow: hidden;
+    text-align: center;
+    white-space: nowrap;
+    background-image: linear-gradient(45deg, hsla(0, 0%, 100%, .15) 25%, transparent 0, transparent 50%, hsla(0, 0%, 100%, .15) 0, hsla(0, 0%, 100%, .15) 75%, transparent 0, transparent);
+    background-size: 1rem 1rem;
+    animation: progress-bar-stripes 1s linear infinite;
+    transition: width .6s ease;
+    border-radius: 4px;
+}
```
Comment (petrKavulok) [inline demo/public/body.css:28]: hm.. I wonder.. what are these styles for?

**Review by petrKavulok — CHANGES_REQUESTED** [review]
Just a few minor things. It looks pretty good!

File: demo/public/body.css (hunk 2 above)
Comment (Pe5h4) [reply demo/public/body.css:28]: @petrKavulok this is the splashscreen loader when app is not fully loaded

File: demo/public/index.html [hunk 3]
```diff
@@ -1,40 +1,49 @@
...
-		<meta property="og:description" content="...">
-	</head>
-	<body>
-		<noscript>
-			You need to enable JavaScript to run this app.
-		</noscript>
-		<div id="app">
-		</div>
-	</body>
+<head>
```
Comment (Pe5h4) [inline demo/public/index.html:3]: @Samsunggora Wrong indentation, please update within this file

File: demo/src/modules/home/containers/HomeContainer.js [hunk 4]
```diff
@@ -1,21 +1,12 @@
...
-	}
+function HomeContainer(props) {
+	return (
+		<Container fluid>
+			<p>Hello world</p>
+		</Container>
+	);
 }
 
 export default HomeContainer;
```
Comment (Pe5h4) [inline demo/src/modules/home/containers/HomeContainer.js:12]: @Samsunggora Add  a newline to the end of the file

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -60,6 +60,8 @@
 
 - Applying lazy loading in About, Auth, Tools and Maintenance modules. (INDIGO Sprint 230428, [!415](https://github.com/TeskaLabs/asab-webui/pull/415))
 
+- Enable dark mode in Home screen, refactor class components to functional, update body.css in demo app, add dynamic configuration section to index.html. (INDIGO Sprint 230428, [!420](https://github.com/TeskaLabs/asab-webui/pull/420))
```
Comment (Pe5h4) [inline CHANGELOG.md:63]: @Samsunggora How did you enabled dark mode on HomeScreen?

**Review by Pe5h4 — APPROVED** [review]
@Samsunggora Alright, so far so good. Please update changelog, since there are some misleading informations (enabling dark mode on homescreen)

File: CHANGELOG.md (hunk 5 above)
Comment (Pe5h4) [reply CHANGELOG.md:63]: @petrKavulok not sure about that. This add a class `loaded` which is used to determine whether the app is loaded or not and display the correct background color (and loader). I think the comment above that is not understandable,cant say but I guess Ari wrote it.

What I think is that by refactoring the class component to functional Egor somehow removed the original background color, which was I guess white

Comment (petrKavulok) [reply CHANGELOG.md:63]: @Pe5h4  Yes, you are right. But I believe, what is meant by 'Enable dark mode in Home Screen,...' is adding this 'loaded' classname which handles correct bg color. 
I think the demo app might not have been properly refactored when we introduced the new design and the dark background color was not functional.

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**